akka-persistence/src/main/scala/akka/persistence/Eventsourced.scala

```diff
@@ -5,6 +5,9 @@
 package akka.persistence
 
 import java.util.concurrent.atomic.AtomicInteger
+import akka.util.Reflect
+
+import scala.UnsupportedOperationException
```

Comment:
import not needed, I'd rather use the `java.lang.UnsupportedOperationException`, but I guess this is just an alias of that